index.html

```diff
@@ -12,19 +12,31 @@
     <title>Naji portfolio</title>
 </head>
 
-<body>
+<body class="mobile">
     <header>
         <div class="navbar flex">
             <a class="logo" href="#"> My logo</a>
             <ul class="links">
-                <li><a href="#">Protfolio</a> </li>
-                <li><a href="#">About</a> </li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#Protfolio">Protfolio</a> </li>
+                <li onclick="mobileMenu()"><a href="#about">About</a> </li>
+                <li><a href="#Contact">Contact</a></li>
```

Comment:
- [ ] Well done on creating the nav menu, but I have some issues raised before the project get approved, take a look at this section below, ⬇️ 

-  the spaces between the close icon and the nav items are not the same as in the original, 
- The `font weight`, `spaces`, and `font-size` of your nav items are not the same as the ones on the original design.

<img width="763" alt="1" src="https://user-images.githubusercontent.com/77843050/210263405-640889a5-8bf1-4882-9189-33ab17d69b2c.png">